Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -176,13 +219,13 @@ void ABenchmarkGymGameModeBase::TickPlayersConnectedCheck(float DeltaSeconds)
 	check(Constants);
 
 	// This test respects the initial delay timer in both native and GDK
-	if (Constants->PlayerCheckMetricDelay.IsReady())
+	if (Constants->PlayerCheckMetricDelay.HasTimerGoneOff())
 	{
 		bHasDonePlayerCheck = true;
 		if (ActivePlayers < RequiredPlayers)
 		{
 			// This log is used by the NFR pipeline to indicate if a client failed to connect
-			NFR_LOG(LogBenchmarkGymGameModeBase, Error, TEXT("A client connection was dropped. Required %d, got %d, num players %d"), RequiredPlayers, ActivePlayers, GetNumPlayers());
+			NFR_LOG(LogBenchmarkGymGameModeBase, Error, TEXT("NFR scenario failed: Client connection dropped. Required %d, got %d, num players %d"), RequiredPlayers, ActivePlayers, GetNumPlayers());
```

Comment:
nit: could we make "NFR scenario failed" a const string that can get used everywhere?

Reply:
Sure! I like it!